2200008/3456.py

```diff
@@ -0,0 +1,11 @@
+# 직사각형의 길이 찾기
+T = int(input())
+for t in range(1, T+1):
+    rec = list(map(int, input().split()))
+    for l in rec:
+        if rec.count(l) == 3:
+            print(f"#{t} {l}")
+            break
+        elif rec.count(l) == 1:
+            print(f"#{t} {l}")
+            break
```

Comment:
좋은 풀이네요! 😄 
```suggestion
        if rec.count(l) == 3 or rec.count(l) == 1:
            print(f"#{t} {l}")
            break
```
`if`문과 `elif` 문 내의 수행하는 코드가 같으므로 하나에 합쳐서 작성하면 좋을 것 같습니다!